### DISCUSSION TITLE
[@types/prosemirror-changeset] Typo in `Span` property `length`

### DISCUSSION THREAD

**Discussion by Nextra** [DefinitelyTyped/DefinitelyTyped#60514] [Issues with a @types package] [answered]
DefinitelyTyped:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c432b1015151bc90947ea2ac3b1a50a86734ad0b/types/prosemirror-changeset/index.d.ts#L13

Library:

https://github.com/ProseMirror/prosemirror-changeset/blob/5c67c039fac9a9a2eab0285864bfa58bdf2e4741/src/change.ts#L6

[a comment by an automated account is not shown]

**Comment by pmairoldi** [comment, marked as answer]
I've fixed it here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60520. Seems like it prosemirror-changeset is written in typescript now so if it has been published I'd suggest you use that instead.

**Reply by Nextra** [reply]
Has not been published yet unfortunately. I'll look out for it, thanks!

**Reply by pmairoldi** [reply]
Alright. Well if you approve my PR then the typing will be published. Thanks!

**Reply by Nextra** [reply]
I wasn't aware that is something I am able to do. It's ready now.
